packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -37,7 +37,17 @@ class BuildAotCommand extends BuildSubCommand {
       )
       ..addFlag('interpreter')
       ..addFlag('quiet', defaultsTo: false)
-      ..addFlag('preview-dart-2', negatable: false);
+      ..addFlag('preview-dart-2', negatable: false)
+      ..addOption('extra-front-end-options',
```

Comment:
The string `extra-font-end-options` is repeated across a number of `.dart` files. Could we declare it as a constant somehere instead? Same for `extra-gen-snapshot-options`.